include/nautilus/nautilus.h

```diff
@@ -23,6 +23,9 @@
 #ifndef __NAUTILUS_H__
 #define __NAUTILUS_H__
 
+// TODO: move this somewhere better
+typedef enum {UNCOND, IF_EARLIER, IF_LATER} nk_timer_condition_t;
```

Comment:
This is an ugly way to move this out of `apic.h`, a better way is needed

Reply:
Sounds like we need a timer.h then include it in every file that complains

Reply:
Probably gonna save this until right before we're ready for merge since it will blow up the Files changed a bit